Added utility code for LinkedList Node and BST Node

araut1 opened the pull request:
Added utility code for LinkedList Node and BST Node, so that solving algorithm problems related to these Data structures is easy.

Let me know of any issues. Happy Building!

araut1 committed:
Added utility code for LinkedList Node and BST Node

Added utility code for LinkedList Node and BST Node, so that solving algorithm problems related to these Data structures is easy.

anku580 commented on ListNode.java:
```diff
@@ -0,0 +1,36 @@
+public class ListNode {
```
Can you please add a folder name `Link List` and then add the ListNode to it.

anku580 commented on TreeNode.java:
```diff
@@ -0,0 +1,18 @@
+public class TreeNode {
```
Can you please add a folder name Trees and then add the TreeNode to it.

anku580 reviewed (changes requested):
Please address the comments which I've added. After that I will merge the PR.

araut1 committed:
Added utility code for LinkedList Node and BST Node

Added utility code for LinkedList Node and BST Node, so that solving algorithm problems related to these Data structures is easy.

Let me know of any issues. Happy Building!

araut1 commented:
Thanks for reviewing. I have made the changes. Let me know if any issues.

anku580 reviewed (approved):
Hey, thank you for contributing. 

All the best :clap:
